Neolution.DotNet.Console/DotNetConsoleException.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Disable warnings that are conflicting with a breaking change from .NET 8 SDK (SYSLIB0051)
+#pragma warning disable S4027
+#pragma warning disable S3925
```

Comment:
instead of suppressing you can simply implement the legacy method marked as obsolete, see https://github.com/TiCodeX/SQLSchemaCompare/blob/master/SQLSchemaCompare.Core/Entities/Exceptions/AccountServiceException.cs#L42

Reply:
Good point, this is fixed